test_runner.py

```diff
@@ -255,13 +251,19 @@ def run_tests(args):
                 )
                 if is_integration_test:
                     for test_flavor in integration_tests_flavors[config_file]:
-                        run_test(test_flavor, full_path)
+                        if args.test == "all" or test_flavor.test_id == args.test:
+                            run_test(test_flavor, full_path, args.output_dir)
```

Comment:
I think we should still assert the `output_dir` specified, otherwise we should error out

Reply:
`output_dir` is a required argument to the python program:
```
parser.add_argument("output_dir")
```
If not provided, the program entry would fail:
```
usage: test_runner.py [-h] [--config_dir CONFIG_DIR] [--test TEST] output_dir
test_runner.py: error: the following arguments are required: output_dir
```